inc/user_templates/parameters/ticketparameters.class.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\User_Templates\Parameters;
+
+use CommonDBTM;
+use Glpi\User_Templates\Parameters\Parameters_Types\ArrayParameter;
+use Glpi\User_Templates\Parameters\Parameters_Types\AttributeParameter;
+use Glpi\User_Templates\Parameters\Parameters_Types\ObjectParameter;
+use Item_Ticket;
+use KnowbaseItem;
+use KnowbaseItem_Item;
+use Location;
+use RequestType;
+use SLA;
+use Ticket;
+use TicketValidation;
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+/**
+ * Parameters for "Ticket" items
+ */
+class TicketParameters extends CommonITILObjectParameters
```

Comment:
Missing parameter classes for Changes and Problems.

Reply:
Fixed.